### PR TITLE
chore: switch to sapphire

### PR DESCRIPTION
**Pull request by kyranet**

### CODE REVIEW

File: src/Alestra.ts [hunk 1]
```diff
@@ -2,15 +2,13 @@ import 'module-alias/register';
...
-AlestraClient.defaultGuildSchema.add('supportChannels', 'TextChannel', { array: true });
+import * as colorette from 'colorette';
 
-const client = new AlestraClient(CLIENT_OPTIONS as KlasaClientOptions);
-client.token = TOKEN;
+inspect.defaultOptions.depth = 1;
+colorette.options.enabled = true;
 
-client.connect().catch((error) => {
-	client.console.error(error);
```
Comment (Favna) [inline src/Alestra.ts:15]:
```
client.token = TOKEN

client.connect()
```

This was still the dumbest and ao-est decision of Klasa Core ever if you ask me. Thank god we don't have that anymore.

File: src/commands/Admin/exec.ts [hunk 2]
```diff
@@ -1,36 +1,33 @@
...
-@ApplyOptions<AlestraCommandOptions>({
+@ApplyOptions<CommandOptions>({
 	aliases: ['execute'],
-	description: (language) => language.get('COMMAND_EXEC_DESCRIPTION'),
-	extendedHelp: (language) => language.get('COMMAND_EXEC_EXTENDED'),
-	guarded: true,
-	permissionLevel: PermissionLevels.BotOwner,
-	usage: '<expression:string>',
-	flagSupport: true
+	description: 'COMMAND_EXEC_DESCRIPTION',
```
Comment (Favna) [inline src/commands/Admin/exec.ts:10]: Replace this with an actual description please. I know it doesn't mean much but this is just ugly.

File: src/commands/Admin/update.ts [hunk 3]
```diff
@@ -1,47 +1,43 @@
...
 })
-export default class extends AlestraCommand {
-	public async run(message: Message, [branch = 'main']: [string?]) {
+export default class UserCommand extends Command {
+	public async run(message: Message, args: Args) {
+		const branch = await args.pick('string').catch(() => 'main');
 		await this.fetch(message, branch);
 		await this.updateDependencies(message);
-		await this.cleanDist(message);
+		if (args.getFlags('fullRebuild')) await this.cleanDist(message);
```
Comment (Favna) [inline src/commands/Admin/update.ts:22]:
```suggestion
		if (args.getFlags('clean')) await this.cleanDist(message);
```

Lets match Skyra's flags

**Review by Favna — CHANGES_REQUESTED** [review]

**Comment by kyranet** [comment]
With ca1cf8d, the banner looks like this:

image

**Comment by kyranet** [comment]
Screenshots from VSCode (with a small enhancement).

## Production mode:
image

## Development mode:
image

**Review by Favna — APPROVED** [review]

**Pull request merged**